app/src/gui/fields/maps/MapFormField.tsx

```diff
@@ -121,10 +135,27 @@ export function MapFormField({
           callbackFn={mapCallback}
           geoTiff={props.geoTiff}
           projection={props.projection}
+          setNoPermission={setNoPermission}
         />
-        {isAlert && (
+        {noPermission && Capacitor.getPlatform() === 'web' && (
+          <Alert severity="error" sx={{width: '100%'}}>
+            Please enable location permissions for this page. In your browser,
+            look to the left of the web address bar for a button that gives
+            access to browser settings for this page.
+          </Alert>
+        )}
+        {noPermission && Capacitor.getPlatform() === 'android' && (
+          <Alert severity="error" sx={{width: '100%'}}>
+            Please enable location permissions for {APP_NAME}. Go to your device
+            Settings &gt; Apps &gt; {APP_NAME} &gt; Permissions &gt; Location
+            and select "Allow all the time" or "Allow only while using the app".
+          </Alert>
+        )}
+        {noPermission && Capacitor.getPlatform() === 'ios' && (
           <Alert severity="error" sx={{width: '100%'}}>
-            Please enable location permissions for this app.
+            Please enable location permissions for {APP_NAME}. Go to your device
+            Settings &gt; Privacy & Security &gt; Location Services &gt;
+            {APP_NAME} and select "While Using the App".
           </Alert>
         )}
```

Comment:
same comment here re: repetition - are these the same? Maybe even refactor the message into utils function or something.